src/containers/Main/reducers.js

```diff
@@ -22,9 +32,6 @@ const ACTION_HANDLERS = {
       aq: action.response.breezometer_aqi,
```

Comment:
Instead of unshift you could use, something like
```  
    const { response: { country_name, breezometer_aqi, breezometer_color } } = action;
    const newData = {
      location: country_name,
      aq: breezometer_aqi,
      color: breezometer_color,
    };
    return {
      error: '',
      isFetching: false,
      data: [newData, ...state.data],
    };
```